docs/reference/ml/df-analytics/apis/dfanalyticsresources.asciidoc

```diff
@@ -124,4 +123,67 @@ An {oldetection} configuration object has the following properties:
 `feature_influence_threshold`:: 
   (double) The minimum {olscore} that a document needs to have in order to 
   calculate its {fiscore}. 
-  Value range: 0-1 (`0.1` by default).
\ No newline at end of file
+  Value range: 0-1 (`0.1` by default).
+  
+
+[discrete]
+[[regression-resources]]
+==== {regression-cap} configuration objects
+
+A `{regression}` configuration object has the following properties:
+
+`dependent_variable`::
+  (Required, string) Defines which field of the {dataframe} is to be predicted. 
+  The data type of the field must be numeric. It is also known as continuous 
+  target variable.
+
+`eta`::
+  (Optional, double) The shrinkage applied to the weights. 
+
+`feature_bag_fraction`::
+  (Optional, double) Defines the fraction of features which will be used when 
+  selecting a random bag for each candidate split. 
+
+`gamma`::
+  (Optional, double) A regularization parameter associated with tree training. 
+  Defines a linear penalty associated with tree size. A higher value means 
+  training will prefer smaller trees. It aims to prevent overfitting.
+  
+`lambda`::
+  (Optional, double) A regularization parameters associated with tree training. 
+  It is the L2 regularization term which applies to leaf weights of the 
+  individual trees in the forest. It aims to prevent overfitting.
+  
+`maximum_number_trees`::
+  (Optional, integer) Defines the maximum number of trees the forest is 
+  allowed to contain. The maximum value is 2000.
+  
+`prediction_field_name`::
+  (Optional, string) Defines the name of the prediction field in the results. 
+  Defaults to `<dependent_variable>_prediction`.
+  
+`training_percent`::
+  (Optional, integer) Defines what percentage of the data rows that are 
+  eligible for training. Defaults to `100`.
+
+
+[source,js]
```

Comment:
Should we use wine quality as an example? Could we stick with the house prices example?

Reply:
I personally am indifferent, but using the wine quality dataset as an example for supervised machine learning is very common.